apps/design-system-docs/README.md

```diff
@@ -0,0 +1,25 @@
+# Automattic Design System Website
+
+This website is built using [Docusaurus](https://docusaurus.io/).
+
```

Comment:
We're missing a second heading here, linting will complain because we jump from h1 to h3.

Reply:
Fixed in e9b893a4e6c9d83445eab6050924f7b1586e3c5b